charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -45,6 +45,9 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
     verbs: ["get", "watch", "list"]
+  - apiGroups: ["apiextensions.k8s.io"]
```

Comment:
Why do they need LIST/WATCH on the CRDs? Is this because they inject the certificate with the knative webhook?